index.js

```diff
@@ -19,17 +19,21 @@ var persistState = function (paths, config) {
   return function (store) {
     var initialized = false
 
-    store.on('@init', function () {
+    store.on('data/update', function (_, data) {
+      return data
+    })
+
+    store.on('@init', async function () {
       initialized = true
 
       try {
-        var savedState = storage.getItem(key)
+        var savedState = await storage.getItem(key)
```

Comment:
When I am adding this `await` then store doesn't update.
@ai Can you please help me?